sdk-base/build.gradle.kts

```diff
@@ -29,7 +28,12 @@ dependencies {
     api(project(":common"))
   } else {
     api(Dependencies.mapboxGlNative)
-    api(Dependencies.mapboxCoreCommon)
+    api("com.mapbox.common:common") {
+      version {
+        strictly("18.0.0-SNAPSHOT")
+      }
+      because("Use snapshot in order to support testing core telemetry")
+    }
```

Comment:
@nagineni this is the change that forces to use the snapshot version of common sdk.